hibernate-core/src/main/java/org/hibernate/bytecode/enhance/internal/bytebuddy/InlineDirtyCheckingHandler.java

```diff
@@ -58,7 +63,7 @@ else if ( !persistentField.hasAnnotation( Id.class )
 					&& !( persistentField.getType().asErasure().isAssignableTo( Collection.class )
 					&& enhancementContext.isMappedCollection( persistentField ) ) ) {
 				implementation = new InlineDirtyCheckingHandler( implementation, managedCtClass,
-						persistentField.asDefined() );
+						persistentField.asDefined(), enhancementContext.isLazyLoadable(persistentField) );
```

Comment:
It would be great to comply to Hibernate coding format aggressively in non-testing code, including:
* use `final` keyword for temporary variable within method as much as possible;
* use space delimiter after `(` and before `)'.